stream/src/placeholder/instance.rs

```diff
@@ -14,6 +18,7 @@ use chacha20poly1305::{
 use commonware_cryptography::{PublicKey, Scheme};
 use commonware_macros::select;
 use commonware_runtime::{Clock, Sink, Spawner, Stream};
+use commonware_utils::SystemTimeExt as _;
```

Comment:
nit: If there is no conflict with the `SystemTimeExt`, it is not necessary to cast `as _`;